makefiles/Mf-embed.in

```diff
@@ -0,0 +1,29 @@
+# Mf-embed.in
+# Copyright 1984-2016 Cisco Systems, Inc.
+# 
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# 
+# http://www.apache.org/licenses/LICENSE-2.0
+# 
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+###############################################################################
+#               the following variables are set up by configure               #
+###############################################################################
+
+m=none
+
+InstallPrefix=/usr
+InstallLib=${InstallPrefix}/lib
+Version=csv9.4.1
+InstallLibBin=${InstallLib}/${Version}/$m
+
+ChezCFLAGS= -I ${InstallLibBin}
+ChezLDFLAGS= -L ${InstallLibBin}
+ChezLDLIBS= -Wl,--push-state -Wl,-Bsymbolic -lchezscheme -Wl,--pop-state  -lm -ldl -lncurses -lrt
```

Comment:
`ChezLDLIBS` does not contain `-lpthread` on my system, despite being a `--threaded` x86_64 Linux build, but it probably should, I think?